bin/openstudio_meta

```diff
@@ -62,6 +74,28 @@ def parse_options(opts = nil, argv = nil)
   # Default opts to a blank optionparser if none is given
   opts ||= ::OptionParser.new
 
+  opts.on('--ruby-lib-path PATH', 'Additional Ruby path, to locate openstudio') do |path|
+    ENV['RUBYLIB'] = path
+  end
+
+  require 'tmpdir'
+  require 'fileutils'
+  ENV['OS_SERVER_LOG_PATH'] = Dir.mktmpdir
+  opts.on('--server-log-path [PATH]', 'Path to server logs') do |path|
+    FileUtils.remove_entry ENV['OS_SERVER_LOG_PATH']
+    ENV['OS_SERVER_LOG_PATH'] = path
+  end
+
+  ENV['RAILS_ENV'] = 'local'
+  opts.on('--rails-env [ENV]', 'Rails Environment') do |env|
+    ENV['RAILS_ENV'] = env
+  end
+
+  ENV['OS_SERVER_HOST_URL'] = 'http://localhost:8080'
```

Comment:
I would prefer to just let that dynamic thing do its work, but from empirical testing I found that if OS_SERVER_HOST_URL is not set in the environment then the install gems command fails. 

Granted I deduced that before you made you your last commit, so things might have changed.

If the port is selected randomly, how does PAT find out where to connect?